src/framework/extensions/internal/extensionsprovider.cpp

```diff
@@ -183,7 +183,7 @@ mu::Ret ExtensionsProvider::run(const Action& a)
     //! TODO Add check of type
 
     Ret ret;
-    if (a.apiversion == 1) {
```

Comment:
hmm, this version seems better to me, why did you decide to change it?

Reply:
I have with more than 2 combinations, the combinations are:
* Old qml plugin and Api v1 (global curScore)
* New extension and Api v1 (global curScore) (I plan port old plugins to extensions, add manifest...)
* New extension and new Api (global api.xxx)

thus, api v1 is used in 2 different configurations, which I need to distinguish, because they are launched differently